tests/ci/.gitignore

```diff
@@ -1,3 +1,4 @@
 *_lambda/lambda-venv
 *_lambda/lambda-package
 *_lambda/lambda-package.zip
+gh_cache
```

Comment:
Do you use vscode for such files? https://stackoverflow.com/questions/44704968/visual-studio-code-insert-newline-at-the-end-of-files

Reply:
Yes, I use vscode, thanks!